.github/workflows/delete-artifact.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Main function to handle documentation preview comments
+ * @param {object} params - Parameters object containing context and github
+ * @param {object} params.github - GitHub API client
+ * @param {object} params.context - GitHub context
+ * @param {object} params.env - Environment variables
+ */
+module.exports = async ({ github, context, env }) => {
+  const artifactName = env.ARTIFACT_NAME;
+  const runId = env.RUN_ID;
+
+  if (!artifactName || !runId) {
+    throw new Error("Missing required parameters: ARTIFACT_NAME, RUN_ID");
+  }
+
+  const repo = "mlflow/mlflow";
+  const [owner, repoName] = repo.split("/");
+
+  try {
+    // INFO: https://octokit.github.io/rest.js/v22/#actions-list-workflow-run-artifacts
+    const { artifacts } = await github.rest.actions.listWorkflowRunArtifacts({
+      owner,
```

Comment:
```suggestion
    const { data: { artifacts } } = await github.rest.actions.listWorkflowRunArtifacts({
      owner,
```

Reply:
You are absolutely correct: https://octokit.github.io/types.ts/interfaces/OctokitResponse.html